models/yang/sonic/sonic-acl.yang

```diff
@@ -0,0 +1,221 @@
+module sonic-acl {
+	namespace "http://github.com/Azure/sonic-acl";
+	prefix sacl;
+	yang-version 1.1;
+
+	import ietf-inet-types {
+		prefix inet;
+	}
+
+	import sonic-common {
+		prefix cmn;
+	}
+
+	import sonic-port {
+		prefix prt;
+	}
+
+	import sonic-mirror-session {
+		prefix sms;
+	}
+
+	organization
+		"SONiC";
+
+	contact
+		"SONiC";
+
+	description
+		"SONIC ACL";
+
+	revision 2019-05-15 {
+		description
+			"Initial revision.";
+	}
+
+	container sonic-acl {
+
+		container ACL_TABLE {
+
+			list ACL_TABLE_LIST {
+				key "aclname";
+
+				leaf aclname {
+					type string {
+						pattern '[a-zA-Z0-9]{1}([-a-zA-Z0-9_]{0,71})';
+						length 1..72;
+					}
+				}
+
+				leaf policy_desc {
```

Comment:
Same here.

#define TABLE_DESCRIPTION "POLICY_DESC"
#define TABLE_TYPE        "TYPE"
#define TABLE_PORTS       "PORTS"
#define TABLE_SERVICES    "SERVICES"

Are we going  to convert to upper case before writing to DB?

Reply:
If it is upper case always in DB, we can have the YANG node name accordingly.

Reply:
This would need separate discussion outside this review.

Reply:
**Not Applicable. Working fine as per current design in SONiC**